README.md

```diff
@@ -41,96 +73,185 @@ pod "KlaviyoSwift"
+<TODO: Should we talk about what happens to a push token here?>
 
 ```swift
-let event = Event(name: .AddedToCartMetric, properties: [
-    "Total Price": 10.99,
-    "Items": ["Hot Dog", "Fries", "Shake"]
-], identifiers: .init(email: "junior@blob.com"),
-profile: [
-    "$first_name": "Blob",
-    "$last_name": "Jr"
-], value: 10.99)
+// start a profile for Blob Jr.
+let profile = Profile(email: "junior@blob.com",  firstName: "Blob",  lastName: "Jr.")
+KlaviyoSDK().set(profile: profile)
 
-KlaviyoSDK().create(event: event)
+// stop tracking Blob Jr.
+KlaviyoSDK().resetProfile()
+
+// start a profile for Robin Hood
+let profile = Profile(email: "junior@blob.com",  firstName: "Robin",  lastName: "Hood")
+KlaviyoSDK().set(profile: profile)
 ```
+### Anonymous Tracking Notice
 
-### Arguments
+By default, Klaviyo will begin tracking unidentified users in your app when a push token is set or we receive an event to track.
+This means you will be able to track events or set a push notifications token from users in your app without any user information provided.
+When an email or other primary identifier is provided, Klaviyo will merge the data from the anonymous user to a new identified user.
 
-The `create` method takes an event object as an argument. The event can be constructed with the following arguments:
+Prior to version 1.7.0, the Klaviyo SDK used the [Apple identifier for vendor (IDFV)](https://developer.apple.com/documentation/uikit/uidevice/1620059-identifierforvendor) to facilitate anonymous tracking.
+Starting with version 1.7.0, the SDK will use a cached UUID that is generated when the SDK is initialized.
+For existing anonymous profiles using IDFV, the SDK will continue to use IDFV, instead of generating a new UUID.
 
-- `name`: The name of the event you want to track, as a EventName enum. The are a number of commonly used event names provided by default. If you need to log an event with a different name use `CustomEvent` with a string of your choosing. This argument is required to track an event.
+## Event tracking
 
-- `profile`: An dictionary of properties that belong to the person who did the action you're tracking. Including an `$email`, `$phone_number` or `$id` key associates the event with particular profile in Klaviyo. In addition the SDK will retain these properties for use in future sdk calls, therefore if they are not included previously set identifiers will be used to log the event. This argument is optional.
+The SDK provides tools for tracking events that users perform on your app via the [Create Client Event API](https://developers.klaviyo.com/en/reference/create_client_event).
+Below is an example of how to track an event:
 
-- `properties`: An dictionary of properties that are specific to the event. This argument is optional.
+```swift
+// using a predefined event name
+let event = Event(name: .StartedCheckoutMetric,
+                  properties: [
+                        "product.1": "t-shirt.99",
```

Comment:
These properties look a little odd but maybe they are derived from a real event?

Reply:
No I just came up with them. What would you suggest?

Reply:
like name or color or size or something.

Reply:
Ya pretty generic props should be fine here. Pretty sure what we put in the Android one was like that too so can just copy from that if you are unsure

Reply:
That makes more sense. Updated.